friendsofphp/php-cs-fixer/2.2/.php_cs.dist

```diff
@@ -1,8 +1,14 @@
 <?php
 
+$finder = PhpCsFixer\Finder::create()
+    ->in(__DIR__)
+    ->exclude('var')
+;
+
 return PhpCsFixer\Config::create()
     ->setRules([
         '@Symfony' => true,
         'array_syntax' => ['syntax' => 'short'],
```

Comment:
off topic here but @Symfony should soon include short array syntax rule IMO

Reply:
Now it does by default: https://github.com/symfony/symfony/pulls?utf8=%E2%9C%93&q=is%3Apr+is%3Aclosed+%22short+array%22+in%3Atitle+

Reply:
I mean PHP CS Fixer should include short array in the „group“ of fixers in „@ Symfony“